backend/src/test/java/de/neuefische/backend/service/ApplicationServiceTest.java

```diff
@@ -0,0 +1,113 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.dto.ApplicationDTO;
+import de.neuefische.backend.model.ApplicationModel;
+import de.neuefische.backend.repository.ApplicationRepo;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class ApplicationServiceTest {
+
+    private final ApplicationRepo mockRepo = mock(ApplicationRepo.class);
+
+    private final IdService mockIdService = mock(IdService.class);
+
+    @Test
+    void testGetAllApplications() {
```

Comment:
Your tests look good! I would only choose exactly name of each test like getAllApplications_shouldReturnAllApplications etc.